Add missing encoding aliases.  It may be contributed upstream at some point,
but the suitability (or lack thereof) has not yet been determined.

The patch comes from solaris-userland.

--- Python-3.14.6/Lib/encodings/aliases.py.orig
+++ Python-3.14.6/Lib/encodings/aliases.py
@@ -83,6 +83,7 @@
 
     # cp1251 codec
     '1251'               : 'cp1251',
+    'ansi_1251'          : 'cp1251',
     'windows_1251'       : 'cp1251',
 
     # cp1252 codec
@@ -247,6 +248,7 @@
     'u_jis'              : 'euc_jp',
 
     # euc_kr codec
+    '5601'               : 'euc_kr',
     'euckr'              : 'euc_kr',
     'korean'             : 'euc_kr',
     'ksc5601'            : 'euc_kr',
@@ -500,6 +502,7 @@
     'shiftjis'           : 'shift_jis',
     'sjis'               : 'shift_jis',
     's_jis'              : 'shift_jis',
+    'pck'                : 'shift_jis',
 
     # shift_jis_2004 codec
     'shiftjis2004'       : 'shift_jis_2004',
@@ -516,6 +519,7 @@
     'tis_620_0'          : 'tis_620',
     'tis_620_2529_0'     : 'tis_620',
     'tis_620_2529_1'     : 'tis_620',
+    'tis620.2533'        : 'tis_620',
     'iso_ir_166'         : 'tis_620',
 
     # utf_16 codec
